[2025-02-02T18:59:18.533+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T18:59:18.544+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flights_pipeline.check_row_numbers manual__2025-02-02T18:57:41.796357+00:00 [queued]>
[2025-02-02T18:59:18.549+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flights_pipeline.check_row_numbers manual__2025-02-02T18:57:41.796357+00:00 [queued]>
[2025-02-02T18:59:18.552+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2025-02-02T18:59:18.561+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): check_row_numbers> on 2025-02-02 18:57:41.796357+00:00
[2025-02-02T18:59:18.566+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1065) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T18:59:18.567+0000] {standard_task_runner.py:72} INFO - Started process 1130 to run task
[2025-02-02T18:59:18.567+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flights_pipeline', 'check_row_numbers', 'manual__2025-02-02T18:57:41.796357+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/zdag.py', '--cfg-path', '/tmp/tmpprxwnks2']
[2025-02-02T18:59:18.567+0000] {standard_task_runner.py:105} INFO - Job 50: Subtask check_row_numbers
[2025-02-02T18:59:18.597+0000] {task_command.py:467} INFO - Running <TaskInstance: flights_pipeline.check_row_numbers manual__2025-02-02T18:57:41.796357+00:00 [running]> on host d628bd12729b
[2025-02-02T18:59:18.644+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flights_pipeline' AIRFLOW_CTX_TASK_ID='check_row_numbers' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T18:57:41.796357+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T18:57:41.796357+00:00'
[2025-02-02T18:59:18.648+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T18:59:18.694+0000] {logging_mixin.py:190} INFO - Nombre de lignes dans la table : 38874
[2025-02-02T18:59:18.703+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T18:59:18.711+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T18:59:18.713+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flights_pipeline, task_id=check_row_numbers, run_id=manual__2025-02-02T18:57:41.796357+00:00, execution_date=20250202T185741, start_date=20250202T185918, end_date=20250202T185918
[2025-02-02T18:59:18.740+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-02-02T18:59:18.751+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T18:59:18.755+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
